admin/test/test_release.py

```diff
@@ -53,3 +60,604 @@ def test_non_integer_suffix(self):
             u'Non-integer value "X" for "pre". Supplied version 0.1.2preX',
             unicode(exception.exception)
         )
+
+
+class PublishDocsTests(TestCase):
+    """
+    Tests for :func:``publish_docs``.
+    """
+
+    def publish_docs(self, aws,
+                     flocker_version, doc_version, environment):
+        """
+        Call :func:``publish_docs``, interacting with a fake AWS.
+
+        :param FakeAWS aws: Fake AWS to interact with.
+        :param flocker_version: See :py:func:`flocker_version`.
```

Comment:
Sorry if this is just syntax I don't know, but is this not saying to look for `def flocker_version`. If so, I can't find it.